src/narrow-screen/screens/RestaurantScreen.js

```diff
@@ -112,14 +131,14 @@ export default class extends React.Component {
             restaurantName={this.props.restaurantId}
             menu={this.state.dishesByMenu[this.state.selectedMenuIndex]}
           />
-          : 
+          :
           (
-            this.state.error ? 
-            <div>Some error has ocurred. Please try reloading the page.</div> :
-            <div>Loading...</div>
+            this.state.error ?
+            <PageError>There was an error loading this page. Please try reloading the page or contact the Nomi team by filling out a form at dinewithnomi.com</PageError> :
+            <RestaurantNotFound>Restaurant Not Found - Please navigate to a different restaurant.</RestaurantNotFound>
           )
```

Comment:
This part should not be an error yet. The page is actually really LOADING.

Reply:
changed it back, also I think files should be up to date now?